lib/resource-enrichers/knative-service-enricher.js

```diff
@@ -0,0 +1,90 @@
+/*
+ *
+ *  Copyright 2016-2017 Red Hat, Inc, and individual contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+'use strict';
+
+const _ = require('lodash');
+const objectMetadata = require('../definitions/object-metadata');
+
+const baseServiceConfig = {
+  apiVersion: 'serving.knative.dev/v1',
+  kind: 'Service',
+  spec: {} // Should be populated in the resource coming in
+};
+
+// Returns a default knative serving service.
+function defaultService (config) {
+  const serviceConfig = _.merge({}, baseServiceConfig);
```

Comment:
Not sure what Node versions you are targeting, but this can now be more easily done with destructuring.

```js
const serviceConfig = { ...baseServiceConfig }
```

Reply:
I think at the time i created nodeshift,  this wasn't available?  I'll have to compare the 2 "merges" to make sure nothing is different.  

Since most of nodeshift does it this way,  we could make this another PR to update the other spots

Reply:
A separate PR to "modernize" to ES6 is probably the right way to go, yeah.